server/server.go

```diff
@@ -1451,6 +1451,32 @@ func (s *Server) fetchAccount(name string) (*Account, error) {
 	return acc, nil
 }
 
+func (s *Server) enableOCSP() error {
+	opts := s.getOpts()
+	// Check if OCSP Stapling was enabled/disabled globally.
+	oc := opts.OCSPConfig
+	if oc == nil || oc.Mode == OCSPModeNever {
+		return nil
+	}
+	// Start OCSP Stapling for client connections.
+	if config := opts.TLSConfig; config != nil {
+		tc, mon, err := s.NewOCSPMonitor(config)
+		if err != nil {
+			return err
+		}
+		// Check if an OCSP stapling monitor is required for this certificate.
+		if mon != nil {
+			// Override the TLS config with one that follows OCSP.
+			opts.TLSConfig = tc
+			s.startGoRoutine(func() { mon.run() })
+		}
+	}
+	// FIXME: Add support for leafnodes, routes, MQTT, WebSocket
+	s.Noticef("OCSP Stapling enabled")
```

Comment:
This could do with identifying which cert it's for (but for now, just mention that requirement in the FIXME comment)

Reply:
Now it outputs for which port the staples will be fetched: 
```
OCSP Stapling enabled for client connections
```